src/Request.php

```diff
@@ -297,7 +312,7 @@ class Request
     /**
      * Initialize
      *
-     * @param Telegram $telegram
+     * @param  Telegram  $telegram
```

Comment:
We need to agree on some standard here, these things just bloat a PR so much :see_no_evil: 

Looks like most changes in this file are cosmetic, some even unpretty.

Reply:
Errg yes, that was not on purpose... Seems like i hit the auto format in PhpStorm here... I'll clean that up.